opsdroid/loader.py

```diff
@@ -304,6 +311,14 @@ def _load_modules(self, modules_type, modules):
             os.makedirs(DEFAULT_MODULE_DEPS_PATH)
         sys.path.append(DEFAULT_MODULE_DEPS_PATH)
 
+        # entry point group naming scheme: opsdroid_ + module type plural,
+        # eg. "opsdroid_databases"
+        epname = "opsdroid_%ss" % modules_type
```

Comment:
Just for code style consistency could you change this line to use `.format` rather than `%s`. We only use `%s` for logging.